lib/miniz-2.2.0/miniz.c

```diff
@@ -7731,3 +7731,10 @@ mz_bool mz_zip_end(mz_zip_archive *pZip)
 #endif
 
 #endif /*#ifndef MINIZ_NO_ARCHIVE_APIS*/
+
+
+mz_uint8 *tdefl_get_png_row_default( mz_uint8 *pImage, mz_bool flip, int w, int h, int y, int bpl, void *target ) {
```

Comment:
I guess this is not needed by now since we are using @lovyan03 to render PNGs.
Will check if it can go, so then we don't need to keep our own **miniz** version that creates unnecessary conflicts